web/renderer/hooks/useDatabaseDetails.ts

```diff
@@ -15,11 +16,13 @@ export default function useDatabaseDetails(): ReturnType {
   const isDolt = res.data?.doltDatabaseDetails.isDolt ?? false;
   const isPostgres =
     res.data?.doltDatabaseDetails.type === DatabaseType.Postgres;
+  const isMysql = res.data?.doltDatabaseDetails.type === DatabaseType.Mysql;
```

Comment:
You shouldn't need this. If both `isDolt` and `isPostgres` are false then it's a mysql db